plugins/event_notification/lib/model/BatchEventNotificationTemplate.php

```diff
@@ -45,14 +45,35 @@ protected function dispatchJob(kScope $scope, kEventNotificationDispatchJobData
 			{
 				$entryId = $object->getEntryId();
 			}
+
+			$objectType = null;
```

Comment:
better define $objectType outside the if condition "if($scope instanceof kEventScope)"
since we use it outside